models/experimental/functional_unet/demo/demo.py

```diff
@@ -18,9 +18,7 @@
 from models.experimental.functional_unet.unet_utils import create_custom_preprocessor
 
 import time
-import tt_lib as ttl
 import os
-from tt_lib import profiler
```

Comment:
Is this import being used? if so, keep this for now.

Reply:
as it is not used in the file I have removed it